config/services/zerotier.xml

```diff
@@ -0,0 +1,8 @@
+
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>ZeroTier</short>
+  <description>ZeroTier creates secure networks between on-premise, cloud, desktop, and mobile devices.</description>
+  <port protocol="tcp" port="9991"/>
+  <port protocol="udp" port="9991"/>
```

Comment:
This is the wrong port! Docs say `9993/udp`. TCP is not used, AFAICS.

https://docs.zerotier.com/zerotier/troubleshooting/

Reply:
Thanks! Fixed